provider/resource_corp_integration.go

```diff
@@ -25,7 +25,7 @@ func resourceCorpIntegration() *schema.Resource {
 				ForceNew:    true,
 				ValidateFunc: func(val interface{}, key string) (warns []string, errs []error) {
 					if !existsInString(val.(string), "mailingList", "slack", "microsoftTeams") {
-						return nil, []error{fmt.Errorf(`"received type %q is invalid. should be "mailingList", "slack", or "microsoftTeams"`, val.(string))}
+						return nil, []error{fmt.Errorf(`received type %q is invalid. should be "mailingList", "slack", or "microsoftTeams"`, val.(string))}
```

Comment:
I had missed in https://github.com/signalsciences/terraform-provider-sigsci/pull/149